### ISSUE TITLE
AsmJS: SIMD.Bool8x16.extractLane produces a wrong result

### ISSUE THREAD

**Issue by Krovatkin** [Microsoft/ChakraCore#824]
SIMD.Bool8x16.extractLane produces a wrong result while extracting "true" values. 

Repo cmd: 

[el_bug.js.txt](https://github.com/Microsoft/ChakraCore/files/224793/el_bug.js.txt)
```
function asmModule(stdlib, imports) {
    "use asm";

    var b16 = stdlib.SIMD.Bool8x16;
    var b16extractLane = b16.extractLane;

 function testExtractLane() {
 
		var b16const = b16(1, 0, 0, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0);
        return b16extractLane(b16const,6)|0; 
    }

    return {testExtractLane:testExtractLane};
}

var m = asmModule(this, {});

print(m.testExtractLane());
print(m.testExtractLane());
print(m.testExtractLane());
print(m.testExtractLane());
print(m.testExtractLane());


print("PASS");
```

```
ch.exe -asmjs -simdjs -maic:0 -mic:0  el_bug.js
```

Expected Output:
```
Successfully compiled asm.js code
1
1
1
1
1
PASS
```

Actual Output:
```
Successfully compiled asm.js code
0
0
0
0
0
PASS
```

**Comment by Krovatkin** [comment]
We will be submitting the fix shortly.

**Comment by nmostafa** [comment]
@dilijev , this is fixed by https://github.com/Microsoft/ChakraCore/pull/826
can you please close it ?

**Comment by dilijev** [comment]
Closed by #826
